share/revng/test/tests/invalidation/invalidation.c.overrides/CABIFunctionType-ABI.yml.invalidations

```diff
@@ -0,0 +1,5 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+plt___libc_start_main
```

Comment:
Are you actually testing `plt___libc_start_main`?
It seems to me that you're committing whatever output you got, which is not how writing ground truth works.